create-parser.sh

```diff
@@ -0,0 +1,41 @@
+#/bin/sh
+
+name=$1;
+
+if [ "$name" = "" ]
+then
+  echo "Please name your parser (name needs to be in kebab-case)";
```

Comment:
Replace "Please name your parser (name needs to be in kebab-case)" by "Please name your parser. It needs to be in kebabcase."